src/nnet3bin/nnet3-egs-augment-image.cc

```diff
@@ -46,6 +54,15 @@ struct ImageAugmentationConfig {
 
 
   void Register(ParseOptions *po) {
+    po->Register("crop", &crop, "Option to set if the input images are not square."
+                 "If crop is set to true the image is randomly scaled then cropped."
+                 "The scaling is randomly so that the shortest side is"
```

Comment:
Change the last part to `The shortest side is scaled to a random size in the range [crop-scale-min, crop-scale-max]`